supplementary_style_guide/glossary_terms_conventions/product_conventions/eap-conventions/eap-terms-and-usage--w.adoc

```diff
@@ -0,0 +1,58 @@
+= W
+
+[discrete]
+=== WebHTTP - Undertow
+
+*Description* Label used for the undertow subsystem in the management console.  Use this formatting to describe this subsystem in the management console.
+
+*Use it?* Yes
+
+*Incorrect forms*
+
+*See*
+
+[discrete]
+=== Web Services
+
+*Description* Heading format for the webservices subsystem.
+
+*Use it?* Yes
+
+*Incorrect forms* webservices
+
+*See* webservices
+
+[discrete]
+=== webservices
+
+*Description* The JBoss EAP subsystem used to configure the web services provider.
+
+NOTE: Only use this format when describing this subsystem.  Do not use this format when describing web services in general.
```

Comment:
There are two spaces after the first sentence.

Reply:
This is now fixed based on updates to the jboss-eap.adoc file in https://github.com/redhat-documentation/supplementary-style-guide/pull/97.